deploy/stacks/pipeline.py

```diff
@@ -207,11 +207,9 @@ def __init__(
                 'Synth',
                 input=source,
                 build_environment=codebuild.BuildEnvironment(
-                    build_image=codebuild.LinuxBuildImage.AMAZON_LINUX_2_3,
+                    build_image=codebuild.LinuxBuildImage.STANDARD_6_0,
                 ),
                 commands=[
-                    'n 16.15.1',
-                    'yum -y install shadow-utils wget && yum -y install openssl-devel bzip2-devel libffi-devel postgresql-devel',
```

Comment:
Looking into https://github.com/aws/aws-codebuild-docker-images/blob/master/ubuntu/standard/6.0/Dockerfile I couldn't find `shadow-utils` (should be `libvshadow-utils` in Ubuntu) and something to related to `postgre` (since we are deleting `postgresql-devel`). Are they shipped with `STANDARD_6_0`?
I am not sure if we use these tools at all at the moment , but wanted to double check with you that we removed them intentionally and there is no need for something like `apt-get install -y libvshadow-utils` 🙂

Reply:
Tbh I wasn't sure if those are needed as well. This specific CodeBuild is only responsible for running `cdk synth` so they seemed unnecessary. It's worth to double check with @dlpzx though.

On a related note, since I've updated this one, might as well upgrade other CBs to `STANDARD_6_0` but I will wait for the confirmation first.

Reply:
The reason behind installing node directly in CodeBuild is that node16 is needed for CDK2 and it was not available at runtime in `AMAZON_LINUX_2_3` images. With `AMAZON_LINUX_2_4` node16 is installed at runtime so we can remove those lines. At first sight I would prefer to use `AMAZON_LINUX_2_4` instead of Ubuntu to avoid changes that could cause errors (plus documentation changes), but maybe I am missing something.

I am also testing the libraries that we are installing ('yum -y install shadow-utils wget && yum -y install openssl-devel bzip2-devel libffi-devel postgresql-devel') caused by the networking changes imposed in #307.

Reply:
Thanks @dlpzx, makes sense. Let me give `AMAZON_LINUX_2_4` a try. I'll also keep the libraries to reduce blast radius.

Reply:
Upgraded all CBs to `AMAZON_LINUX_2_4` & performed a test deployment - all good.

Reply:
@dlpzx all of the packages are already in 4.0 image so we're good removing them. Removed & ran through the deployment pipeline to verify 👍